Yoast/wordpress-seo
# No longer use .min as a suffix

## Summary


This PR can be summarized in the following changelog entry:

* No longer use `.min` in _all_ our asset URLs, so there's no longer a difference in URLs between development and non-development setups.

## Relevant technical choices:

* Completely remove the suffix code.
* Maintain the `WPSEO_CSSJS_SUFFIX` constant as it's still used in extensions.

## Test instructions

This PR can be tested by following these steps:

For both free and Premium:
* The plugin should still work on all its pages and on edit-post etc, not ever throwing 404s for CSS and JS files.
* Create zips with create_beta.sh + check if they're working correctly (see previous item).

I already created this one (Alexander):
[wordpress-seo-min.zip](https://github.com/Yoast/wordpress-seo/files/4069951/wordpress-seo-min.zip)

* Test with at least one addon that still uses the `WPSEO_CSSJS_SUFFIX`

## Quality assurance


Fixes #14186
